pkg/service/roommanager.go

```diff
@@ -513,11 +513,26 @@ func (r *RoomManager) iceServersForRoom(ri *livekit.Room) []*livekit.ICEServer {
 		})
 	}
 	if r.config.TURN.Enabled {
-		iceServers = append(iceServers, &livekit.ICEServer{
-			Urls:       []string{fmt.Sprintf("turns:%s:443?transport=tcp", r.config.TURN.Domain)},
-			Username:   ri.Name,
-			Credential: ri.TurnPassword,
-		})
+		var urls []string
+		if r.config.TURN.UDPPort > 0 {
+			urls = append(urls, fmt.Sprintf("turn:%s:443?transport=udp", r.config.RTC.NodeIP))
```

Comment:
what is the motivation in reporting both LB port and non-LB port?

I'm wondering if it would confuse clients when we are returning unresponsive hosts.

Reply:
@davidzhao Lol, I just removed it since browsers send request to all urls at the same time. It was for proxy with 2 SNIs so direct traffic is less costly to CPU, but not worthy it to double browser ICE requests.